storage/tests/unit/test_blob.py

```diff
@@ -366,7 +366,7 @@ def test__make_transport(self, fake_session_factory):
 
     def test__get_download_url_with_media_link(self):
         blob_name = 'something.txt'
-        bucket = mock.Mock(spec=[])
+        bucket = _Bucket(name='IRRELEVANT')
```

Comment:
Please look at the `test_blob` module:  it *already* uses a `_Bucket` class almost everywhere (84 places currently), and replacing that with a `_make_blob` helper returning a `mock.Mock` instance would be pointless churn in the context of this PR.

I'm not convinced the "contributors won't understand", given that the `_Bucket` class is used everywhere in the module:  how hard is it to do the "search for this name in this module" bit in any editor since even the original `vi` ('*' or '#' WFM in `vim`).  Centralizing the construction of a `mock.Mock` just means they have to look it up somewhere, anyway.